src/components/Calculator.js

```diff
@@ -1,30 +1,56 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      obj: {
+        total: '0',
+        next: '',
+        operation: '',
+      },
+    };
+    this.sendData = this.sendData.bind(this);
+  }
+
+  sendData = (obj, btnValue) => {
```

Comment:
- [ ] <b>OPTIONAL:</b> It would make more sense if you called this function `handleClick`. Kindly consider it.

Reply:
Hi @rukundoeric thanks a lot for your suggestion, I will fit it 😃  hands-on 👨‍💻